ci-operator/step-registry/gcp/provision/private-dns-zone/gcp-provision-private-dns-zone-ref.yaml

```
@@ -19,9 +19,5 @@ ref:
     default: "yes"
     documentation: |-
       Whether to pre-create the DNS private zone in the interested project.
-  - name: BASE_DOMAIN
-    default: ""
-    documentation: |-
-      The base domain.
   documentation: |-
     The step generates private DNS zone settings. 
```

Update doc description: 
```suggestion
    The step reads baseDomain install-config.yaml and generates private DNS zone settings.
```